### PR TITLE
[REF] web: convert core>browser>title service tests to hoot

### PR DESCRIPTION
**Pull request by SimonGenin**

### CODE REVIEW

**Review by Iucapad — APPROVED** [review]
Good job. @robodoo r+

File: addons/web/static/tests/core/browser/title_service.test.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+});
+
+test("add title part", () => {
+    titleService.setParts({ one: "Odoo", two: null });
+    expect(titleService.current).toBe("Odoo");
+    titleService.setParts({ three: "Import" });
+    expect(titleService.current).toBe("Odoo - Import");
+});
+
+test.debug("modify title part", () => {
```
Comment (aab-odoo) [inline addons/web/static/tests/core/browser/title_service.test.js:30]: debug :-)

Comment (aab-odoo) [reply addons/web/static/tests/core/browser/title_service.test.js:30]: wow, and it's not spotted by the python test. We must fix/add the python test right now. @SimonGenin Could you do it?

Comment (SimonGenin) [reply addons/web/static/tests/core/browser/title_service.test.js:30]: ok, on it

**Review by aab-odoo — APPROVED** [review]
robodoo r+

**Comment by aab-odoo** [comment]
robodoo retry

**Pull request closed without merging**